src/pages/home-page-slides/home-page-slides-box-text-mobile/home-page-slides-box-text-mobile.tsx

```diff
@@ -0,0 +1,41 @@
+import { Dispatch, FC } from "react";
+import {
+  BoxTextMobile,
+  SmallParagraph,
+  MainParagraph,
+  ButtonBoxText,
+} from "./home-page-slides-box-text-mobile.style";
+import ButtonsSlider from "../home-page-slides-button-slider/buttons-slider";
+import { ButtonsBox } from "../home-page-slides-mobile-width.style";
+import { CarouselModel } from "../../../model/model";
+
+interface HomePageSlidesBoxTextMobilePropsModel {
+  idxObj: number;
+  carouselArr: CarouselModel[];
+  setIdxObj:Dispatch<React.SetStateAction<number>>
```

Comment:
idxObj, setIdxObj - what exactly does this mean? If you mean the "active" element of the carousel, you can call it something else, so that it is more understandable.

Reply:
ok, i repare name